src/layouting/data_limits.jl

```diff
@@ -144,6 +144,9 @@ function iterate_transformed(plot)
 end
 
 function iterate_transformed(points, model, space, trans_func)
+    if space != :data
+        tuple()
+    end
```

Comment:
This should apply from the outside via `exclude`

https://github.com/MakieOrg/Makie.jl/blob/ffc31a0e7d21584082443daf21c59227d95ed13e/src/layouting/data_limits.jl#L194-L202

starting from

https://github.com/MakieOrg/Makie.jl/blob/ffc31a0e7d21584082443daf21c59227d95ed13e/src/makielayout/blocks/axis.jl#L859-L882

Scene should already be fine

https://github.com/MakieOrg/Makie.jl/blob/ffc31a0e7d21584082443daf21c59227d95ed13e/src/scenes.jl#L571-L583

Reply:
Oh actually axis should be fine as is too, since it has `is_data_space(to_value(get(plot, :space, :data))) || return true `.